plugins/database_api/include/golos/plugins/database_api/api_objects/owner_authority_history_api_object.hpp

```diff
@@ -7,6 +7,8 @@ namespace golos {
     namespace plugins {
         namespace database_api {
 
+            using protocol::authority;
+            using golos::protocol::account_name_type;
```

Comment:
Why did you add these lines?

Reply:
I moves witness_api_object and feed_history_api_object to other directory, and those files had plenty of using's so after i had to restore some of those to make project buildable